src/backend/api/Fusion.Resources.Api/Notifications/InternalRequestNotification/NotifyRequestCreatorHandler.cs

```diff
@@ -26,7 +26,7 @@ protected override async Task Handle(NotifyRequestCreator request, CancellationT
             {
                 var allocationRequest = await GetInternalRequestAsync(request.RequestId);
 
-                var arguments = new NotificationArguments($"A personnel request has been updated") { AppKey = "personnel-allocation" };
+                var arguments = new NotificationArguments($"A personnel request has been updated (you are notified as creator)") { AppKey = "personnel-allocation" };
```

Comment:
I would consider chaning the language to be more fluent. For example:

A personnel request you created has been updated.